src/timbres.json

```diff
@@ -0,0 +1,86 @@
+{
+    "inharmonicSimple": {
+        "lsys golden dense var1": [
```

Comment:
If you're going use names this long, go all the way: "L-system golden dense A" (and similarly B instead of var2)